tests/rptest/tests/read_replica_e2e_test.py

```diff
@@ -111,6 +130,38 @@ def test_produce_is_forbidden(self, partition_count: int) -> None:
             err_msg="Could not create read replica topic. Most likely " +
             "because topic manifest is not in S3.")
 
+    def _bucket_usage(self) -> BucketUsage:
+        assert self.redpanda and self.redpanda.s3_client
+        keys: set[str] = set()
+        s3 = self.redpanda.s3_client
+        num_objects: int = 0
+        total_bytes: int = 0
+        for o in s3.list_objects(self.s3_bucket_name):
```

Comment:
where is `self.s3_bucket_name` defined?

Reply:
Should have marked this a draft earlier, sorry. The merge conflict broke this file. That field was removed in 0aa5cb08dd7ebd79eba045c9d8fe30a8a38d7f71. I'll fix it up and re-open this PR when its ready.